actions/run_action.go

```diff
@@ -113,11 +117,17 @@ func (run *RunAction) doRun(context debos.DebosContext) error {
 		label = path.Base(run.Script)
 	} else {
 		cmdline = []string{run.Command}
-		commands := strings.Split(run.Command, "\n")
+		commands := strings.Split(strings.TrimSpace(run.Command), "\n")
```

Comment:
would suggest moving the `strings.TrimSpace` to the next line so the label really is trimmed then ?

Reply:
The purpose here was to make sure that LFs at the end of the line don’t increase the number of lines.

Reply:
`Split` splits a single LF-terminated line into an array of two strings, the first being the text of the line and the second an empty string.

Reply:
right, that wasn't obvious till you mentioned it, perhaps a comment would be good ?

Reply:
Done.